src/utils/configs.js

```diff
@@ -22,9 +22,16 @@ let isAdmin = false;
   const el = document.querySelector(`meta[name='env:${x.toLowerCase()}']`);
   configs[x] = el ? el.getAttribute("content") : process.env[x];
 
-  if (x === "BASE_ASSETS_PATH" && configs[x]) {
+  if (x === "BASE_ASSETS_PATH" && configs["BASE_ASSETS_PATH"]) {
     // eslint-disable-next-line no-undef
-    __webpack_public_path__ = configs[x];
+    __webpack_public_path__ = configs["BASE_ASSETS_PATH"];
+
+    // BASE_ASSETS_PATH might be a relative URL like "/" when it is set in
+    // .env or .defaults.env when running locally. We need to convert that
+    // to an absolute URL.
+    if (configs["BASE_ASSETS_PATH"] && !configs["BASE_ASSETS_PATH"].startsWith("http")) {
+      configs["BASE_ASSETS_PATH"] = new URL(configs["BASE_ASSETS_PATH"], window.location);
```

Comment:
Nit (up to you) but since it's only one if check above `!configs[BASE_ASSETS_PATH]`, I'd put assigning configs[BASE_ASSETS_PATH] as a ternary.

Reply:
IMO the if statement is more readable in this case, so I'll leave it as is.